pkg/user/signer_test.go

```diff
@@ -69,12 +71,44 @@ func (s *SignerTestSuite) TestSubmitTx() {
 	require.EqualValues(t, 0, resp.Code)
 }
 
-func (s *SignerTestSuite) ConfirmTxTimeout() {
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+// TestConfirmTx verifies that the ConfirmTx method returns the expected results.
```

Comment:
In my opinion, "the expected results" is vague. If this comment doesn't provide value, let's remove it entirely.

```suggestion
```